lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.5.0-dev",
+  "version": "3.5.0-dev.1",
```

Comment:
This change was required as lerna was not able to detect private client changes in the PR since this branch uses another feature branch as base. This resulted in the generator creating packages using the older versions.

Reply:
Once we merge the feature branch, we will no longer need this version change, correct?

Reply:
Yup! We'll revert it once it's merged into feature branch before we merge that into develop.